modular_skyrat/modules/plushiebox/code/game/objects/items/miscellaneous.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/choice_beacon/box/plushie
+	name = "choice beacon (plushie)"
+	desc = "Summons a plushie with this beacon"
+	icon_state = "gangtool-blue"
+
+/obj/item/choice_beacon/box/spawn_option(choice,mob/living/M)
+	..()
+
+/obj/item/choice_beacon/box/plushie/generate_display_names() // Lol, I'm not touching that.
+	var/list/plushie_list = list()
+	//plushie list used for plushies, the other list is used to blacklist. It do work though.
+	var/list/more_plushies_list = subtypesof(/obj/item/toy/plush) - list(/obj/item/toy/plush/narplush, /obj/item/toy/plush/awakenedplushie, /obj/item/toy/plush/ratplush)
```

Comment:
this could be done better for future code support at the cost of minor modularity loss; add a 
CAN_BE_BEACONED var or something like that, then check it and see if you should add it to the list of selectable items, maintainability issue.

Reply:
Yeah some plushies do bad stuff.